content/models.py

```diff
@@ -3,27 +3,49 @@
 
 
 class OurStory(models.Model):
-    img = models.ImageField(blank=True, null=True)
-    text = models.TextField()
-    color = models.CharField(max_length=20, blank=True)
-    order = models.PositiveIntegerField()
+	img = models.ImageField(upload_to='aboutus', blank=True, null=True)
+	text = models.TextField()
+	color = models.CharField(max_length=20, blank=True)
+	order = models.PositiveIntegerField()
 
 
 
-    def __str__(self):
-        return self.text
+	def __str__(self):
+		return self.text
 
 
 class Ourteam(models.Model):
```

Comment:
Feel free to ignore since this isint part of the PR, but maybe in another PR change the name to `OurTeam` instead of `Ourteam` to follow the convention you have set already?